lms/djangoapps/ccx/urls.py

```diff
@@ -25,6 +25,8 @@
     url(r'^ccx_gradebook/(?P<offset>[0-9]+)$',
         'ccx.views.ccx_gradebook', name='ccx_gradebook'),
 
+    url(r'^rename_ccx$', 'ccx.views.rename_ccx', name='rename_ccx'),
```

Comment:
@giocalitri instead of creating a new URL for renaming a CCX, should we try to use a REST API PATCH ?